tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -107,43 +107,6 @@ public async Task WithContainerRegistryUpdatesContainerImageAnnotationsDuringPub
         Assert.Equal(expectedManifest, redisManifest.ToString());
     }
 
-    [Fact]
-    public void EnsureExecutablesWithDockerfileProduceDockerfilev0Manifest()
```

Comment:
Why is this test being deleted? Can it just be updated for the new behavior?

Reply:
This entire file should be deleted except for the uber test. This was before we had split tests for each resource type.